src/libutil/experimental-features.cc

```diff
@@ -268,6 +268,20 @@ constexpr std::array<ExperimentalFeatureDetails, numXpFeatures> xpFeatureDetails
             Allow the use of the [`mounted SSH store`](@docroot@/command-ref/new-cli/nix3-help-stores.html#experimental-ssh-store-with-filesytem-mounted).
         )",
     },
+    {
+        .tag = Xp::ExternalGCDaemon,
+        .name = "external-gc-daemon",
+        .description = R"(
+            Make the garbage collector use an external daemon for the tracing.
+
+            This makes it possible to run a multi-user Nix daemon as a non-root
+            user (only the tracing daemon needs to be root), reducing the attack
+            surface a lot.
```

Comment:
```suggestion
            This makes it possible to run a multi-user Nix daemon as a non-root
            user. Only the tracing daemon needs to be root. This reduces the attack
            surface.
```

No need for subjective statements or parentheses.